src/webpack/plugin/applyDefaults.js

```diff
@@ -0,0 +1,62 @@
+const startCase = require('lodash.startcase');
+const fs = require('fs');
+const path = require('path');
+
+const npmPackage = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'package.json')));
+
+const applyDefaults = ({
+  primaryNavigationItems = [{
+    path: '/home',
+    label: 'Home',
+    contentExtension: 'home',
+    additionalContent: [
+      {
+        label: 'Home',
+        filePath: path.resolve(process.cwd(), 'README.md'),
+      },
+    ],
+  }, {
+    path: '/components',
+    label: 'Components',
+    contentExtension: 'doc',
+  }, {
+    path: '/tests',
+    label: 'Tests',
+    contentExtension: 'test',
+  }],
```

Comment:
Was there talk of breaking tests out into their own site? Assume this is here to maitian what exists today?

Reply:
yep, people could create a specific webpack config with reduced terra-dev-site config to only generate what is being tested. I think this config is up to the person and not terra-dev-site.